src/main/java/iudx/resource/server/metering/readpg/LimitOffSet.java

```diff
@@ -0,0 +1,34 @@
+package iudx.resource.server.metering.readpg;
+
+import io.vertx.core.json.JsonObject;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import static iudx.resource.server.apiserver.util.Constants.LIMITPARAM;
+import static iudx.resource.server.apiserver.util.Constants.OFFSETPARAM;
+import static iudx.resource.server.metering.util.Constants.LIMIT_QUERY;
+import static iudx.resource.server.metering.util.Constants.OFFSET_QUERY;
+
+public class LimitOffSet {
+    private static final Logger LOGGER = LogManager.getLogger(LimitOffSet.class);
+    JsonObject jsonObject = null;
+    StringBuilder finalQuery = null;
+    String limit = null;
+    String offset = null;
+    LimitOffSet(JsonObject jO, StringBuilder q) {
```

Comment:
Change constructor to `LimitOffSet(int limit,int offset)` . only pass info to class what is necessary no need for extra info in JsonObject

Reply:
Here JsonObject have only required details, so i think it already solved issue. 
Please refer ReadQueryBuilder.java

Reply:
Why are wrapping your integers in JsonObjects? Is there any specific need for wrapping integers inside JsonObjects? It will be better to pass integers to class constructors without wrapping them inside JsonObject.

Reply:
Converted to int, changed constructor.